src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatResponseUpdate.cs

```diff
@@ -58,6 +58,30 @@ public ChatResponseUpdate(ChatRole? role, IList<AIContent>? contents)
         _contents = contents;
     }
 
+    /// <summary>
+    /// Creates a new ChatResponseUpdate instance that is a copy of the current object.
+    /// </summary>
+    /// <remarks>The cloned object is a shallow copy; reference-type properties will reference the same
+    /// objects as the original. Use this method to duplicate the response update for further modification without
+    /// affecting the original instance.</remarks>
+    /// <returns>A new ChatResponseUpdate object with the same property values as the current instance.</returns>
+    [Experimental("MEAI001")]
+    public ChatResponseUpdate Clone() =>
```

Comment:
If we have a need for this, I don't think it needs to be experimental; this matches the shape we use elsewhere, so it's unlikely feedback is going to influence it. The most I could imagine us changing would be making it virtual, which we could do in the future (technically that can lead to issues where overrides aren't called from existing binaries that are using call rather than callvirt, but that's a very niche concern).